tests/vmi_gpu_test.go

```diff
@@ -54,7 +54,7 @@ func checkGPUDevice(vmi *v1.VirtualMachineInstance, gpuName string) {
 	Expect(err).ToNot(HaveOccurred(), "GPU device %q was not found in the VMI %s within the given timeout", gpuName, vmi.Name)
 }
 
-var _ = Describe("[Serial][sig-compute]GPU", func() {
+var _ = Describe("[Serial][sig-compute]GPU", Label("needs-gpu"), func() {
```

Comment:
@xpivarc we might need to add the same label to the `MediatedDevices` tests, no?